bait-files/src/main/java/com/google/sps/servlets/UsernameTakenServlet.java

```diff
@@ -33,7 +33,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     response.setContentType("application/json");
     if (username == null || username.equals("")) {
-      String json = new Gson().toJson(true);
+      boolean usernameTaken = true;
+      String json = new Gson().toJson(usernameTaken);
```

Comment:
What's the point of making a variable if it's always going to be true?

Reply:
Removed (was using because it made a test pass before dependency injection)